lib/couchdb_to_sql/table_deleted_marker.rb

```diff
@@ -9,9 +9,8 @@ module CouchdbToSql
   class TableDeletedMarker < TableBuilder
     def execute
       dataset = handler.database[table_name]
-
-      if attributes.key?(:id)
-        handler.changes.log_info "Deletion with 'id' field present (#{primary_key} '#{handler.id}'), assuming tombstone. " \
+      unless doc['_plaque']
```

Comment:
`unless/else` is generally seen as a bad pattern, because it's unnecessarily hard to follow. Please either go with `if !foo` or `if foo` and re-arrange the conditional blocks.